src/Mod/Import/App/ImportOCAF2.cpp

```diff
@@ -450,12 +435,24 @@ bool ImportOCAF2::createObject(App::Document *doc, TDF_Label label,
         feature->Shape.setValue(shape);
         // feature->Visibility.setValue(false);
     }
-    applyFaceColors(feature,{info.faceColor});
-    applyEdgeColors(feature,{info.edgeColor});
```

Comment:
Please check out [here](https://github.com/FreeCAD/FreeCAD/blob/950a569b44c5220a456c49e0455ba54045c03288/src/Mod/Import/Gui/AppImportGuiPy.cpp#L309) for different treatment of array with single and multiple elements.